### ISSUE TITLE
Will you bring vue 3 update in your package?

### ISSUE THREAD

**Issue by mdhasanpatwary** [lucaspulliese/vue-cool-lightbox#119]

**Comment by davision** [comment]
Looking forward to it too!

**Comment by netdjw** [comment]
+1

**Comment by svifty7** [comment]
+1

**Comment by itpropro** [comment]
I don’t think you can expect more updates here. There are (at least on GitHub) no current feature updates or releases, no active changelog, no TypeScript support, no docs in this repo and no composables or Vue/Nuxt 3 support.
So if you are looking for a lightbox module for a current Vue project, I would either create your own lightbox component or check out vue-easy-lightbox.
